cmd/mattermost/commands/channel.go

```diff
@@ -533,3 +546,62 @@ func renameChannelCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func searchChannelCmdF(command *cobra.Command, args []string) error {
+
+	var (
+		aErr      *model.AppError
+		teamFound bool
+		channel   *model.Channel
+		teamId    string
+	)
+
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return errors.Wrap(err, "failed to InitDBCommandContextCobra")
+	}
+	defer a.Shutdown()
+
+	teams, aErr := a.GetAllTeams()
```

Comment:
I don't want to get all the teams, all the time, I prefer to get the team by name if you pass the team name, and get all the teams if not.